frontend/packages/gitops-plugin/src/components/details/GitOpsDetailsPageHeading.tsx

```diff
@@ -33,32 +32,18 @@ const GitOpsDetailsPageHeading: React.FC<GitOpsDetailsPageHeadingProps> = ({
   return (
     <div className="odc-gitops-details-page-heading co-m-nav-title co-m-nav-title--breadcrumbs">
       <BreadCrumbs breadcrumbs={breadcrumbs} />
-      <h1 className="co-m-pane__heading">
+      <h1 className="co-m-pane__heading" style={{ marginRight: 'var(--pf-global--spacer--sm)' }}>
         <div className="co-m-pane__name co-resource-item">
-          <ResourceIcon kind="application" className="co-m-resource-icon--lg" />
           <span className="co-resource-item__resource-name">{appName}</span>
         </div>
         {badge && <span className="co-m-pane__heading-badge">{badge}</span>}
       </h1>
-      <Split className="odc-gitops-details-page-heading__repo" hasGutter>
-        <SplitItem>{t('gitops-plugin~Manifest file repo')}:</SplitItem>
-        <SplitItem isFilled>
-          <Label
-            style={{ fontSize: '12px' }}
-            color="blue"
-            icon={routeDecoratorIcon(manifestURL, 12, t)}
-          >
-            <a
-              style={{ color: 'var(--pf-c-label__content--Color)' }}
-              href={manifestURL}
-              rel="noopener noreferrer"
-              target="_blank"
-            >
-              {manifestURL}
-            </a>
-          </Label>
-        </SplitItem>
-      </Split>
+      <ExternalLink href={manifestURL} additionalClassName={'co-break-all'}>
+        <span style={{ marginRight: 'var(--pf-global--spacer--xs)' }}>
+          {routeDecoratorIcon(manifestURL, 12, t)}
+        </span>
+        <span style={{ marginRight: 'var(--pf-global--spacer--xs)' }}>{manifestURL}</span>
```

Comment:
Are you intentionally adding a margin to the right of this one too?

Reply:
Missed this- changed these to &nbsp; since I was just trying to add space between them